c2rust-analyze/src/main.rs

```diff
@@ -482,6 +533,22 @@ fn run(tcx: TyCtxt) {
         }
     }
 
+    // FIX the fields of structs mentioned in extern blocks
```

Comment:
Are `extern`-block `fn` signatures already `FIXED` elsewhere, or does that still need to be added (in this PR or elsewhere)?

Reply:
I didn't think they needed to be marked fixed because I thought only body-owners were visited?

Reply:
Ah - I don't actually remember whether non-body-owners get visited by `rewrite::ty`, but you might be right.  Either way, I think the args/returns should be marked `FIXED` so that `rewrite::expr` on callers comes up with the right `TypeDesc`s for the callee's args and generates the right casts.

Reply:
I'd like to do that in another PR

Reply:
Created an issue #965